openpype/hosts/aftereffects/api/ws_stub.py

```diff
@@ -315,7 +316,8 @@ def import_file_with_dialog(self, path, item_name):
         res = self.websocketserver.call(
             self.client.call('AfterEffects.import_file_with_dialog',
                              path=path,
-                             item_name=item_name)
+                             item_name=item_name,
```

Comment:
fps should be added in the `import_options`, not as a new argument of the function